pkg/resources/database_role.go

```diff
@@ -21,7 +21,6 @@ var databaseRoleSchema = map[string]*schema.Schema{
 	"name": {
 		Type:             schema.TypeString,
 		Required:         true,
-		ForceNew:         true,
 		Description:      "Specifies the identifier for the database role.",
```

Comment:
Use `blocklistedCharactersFieldDescription`

Reply:
Hmm, I thought it was introduced in ur pr because I was looking for it, but could find it. I'll add it.